packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -34,6 +34,9 @@ export interface ITagInputProps extends IProps {
     /** React props to pass to the `<input>` element */
     inputProps?: HTMLInputProps;
 
+    /** Controlled value of the `<input>` element */
```

Comment:
add another sentence: "This is shorthand for `inputProps={{ value }}`."
also trailing period (and on `inputProps` above too please)

Reply:
Done